src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_search_objects.py

```diff
@@ -3,14 +3,19 @@
 from pcapi import settings
 from pcapi.core import search
 from pcapi.models import Offer
+from pcapi.models import Venue
 
 
 logger = logging.getLogger(__name__)
 
 
-def create_industrial_algolia_indexed_objects() -> None:
+def create_industrial_search_indexed_objects() -> None:
     if settings.ALGOLIA_TRIGGER_INDEXATION:
         logger.info("create_industrial_algolia_objects")
         offer_ids = Offer.query.with_entities(Offer.id).all()
```

Comment:
Ça a l'air de fonctionner (avec un peu de chance, à mon avis !), mais ici on ne récupère pas une liste d'identifiants, mais une liste de tuples à 1 élément :

```
>>> Offer.query.with_entities(Offer.id).all()
[(740,), (741,), (742,), (743,), (744,), ... ]
```

Il faudrait plutôt écrire :

```suggestion
        offer_ids = [offer_id for offer_id, in Offer.query.with_entities(Offer.id).all()]
```

... et idem pour `venue_ids` plus bas.

Reply:
Arf, bien vu. J'ai été trop rapide.
Je corrige (merci).